dummy/src/toga_dummy/widgets/canvas.py

```diff
@@ -245,9 +244,9 @@ def measure_text(self, text, font):
     # Image
 
     def get_image_data(self):
-        """Return a dummy uuid string as the "native" image."""
+        """Return the Toga logo as the "native" image."""
         self._action("get image data")
-        return uuid.uuid4()
+        return (toga.App.app.paths.app / "resources/toga.png").read_bytes()
```

Comment:
It might be better to include a resource in the dummy app, rather than rely on the app path being established.